src/PolygonDraw/Polygon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Polygon as LeafletPolygon } from 'react-leaflet';
+
+import { Coordinate } from 'types';
+import { createLeafletLatLngFromCoordinate } from '../helpers';
+import { MAP } from '../constants';
+
+interface Props {
+    coordinates: Coordinate[];
+    isActive: boolean;
+    onClick: () => void;
+    onMouseEnter: () => void;
+    onMouseLeave: () => void;
+}
+
+export const Polygon: React.FC<Props> = ({ isActive, coordinates, onClick, onMouseEnter, onMouseLeave }) => (
```

Comment:
change: For purpose of consistency , could you please use React.FunctionalComponent or Rect.FC, thanks.

Reply:
Fixed, thanks. 👍

Reply:
Not that this matters at all ;) but I always go for FC 🔥